test/fn/move-contacts.spec.js

```diff
@@ -0,0 +1,264 @@
+const { assert, expect } = require('chai');
+const rewire = require('rewire');
+
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-memory'));
+PouchDB.plugin(require('pouchdb-mapreduce'));
+
+const moveContactsModule = rewire('../../src/fn/move-contacts');
+const updateLineagesAndStage = moveContactsModule.__get__('updateLineagesAndStage');
+const { mockReport, mockHierarchy, parentsToLineage } = require('../mock-hierarchies');
+
+// contacts_by_place is new in v3.0
+const contacts_by_place = {
```

Comment:
I'm concerned these views will change in the future and this script will break. Would be great to somehow have a unit test failure if that happens...